src/java.base/share/classes/java/time/chrono/IsoChronology.java

```diff
@@ -677,6 +677,18 @@ public Period period(int years, int months, int days) {
         return Period.of(years, months, days);
     }
 
+    //-----------------------------------------------------------------------
+    /**
+     * {@inheritDoc}
```

Comment:
I would rather see the statement indicating that ISOChronology returns true; not a generic sentence.
(For each of the Chronologies).

"IsoChronology supports ISO based fields, such as DAY_OF_QUARTER and QUARTER_OF_YEAR."